gui/mozregui/global_prefs.py

```diff
@@ -54,6 +55,23 @@ def __init__(self, parent=None):
         self.ui.http_timeout.setValue(options['http_timeout'])
         self.ui.persist_size_limit.setValue(options['persist_size_limit'])
         self.ui.bg_downloads.setChecked(options['background_downloads'])
+        self.ui.advanced_options.setText(QString("Show Advanced Options"))
```

Comment:
You can use python strings, no need to call QString(). Could you change that ? I guess you won't need the QString import after that. (This is mainly to be consistent with the rest of the code, and to not add unnecessary code).
Sorry I missed that the first time :)